packages/core/src/lib/TwaManifest.ts

```diff
@@ -315,6 +315,139 @@ export class TwaManifest {
     const json = JSON.parse((await fs.promises.readFile(fileName)).toString());
     return new TwaManifest(json);
   }
+
+  /**
+   * Merges the Twa Manifest with the web manifest. Ignores the specified fields.
+   *
+   * @param {string[]} fieldsToIgnore the fields which needs to be ignored.
+   * @param {URL} webManifestUrl the URL where the webmanifest is available.
+   * @param {WebManifest} webManifest the Web Manifest, used as a base for the update of
+   *    the TWA Manifest.
+   * @param {TwaManifest} oldTwaManifest current Twa Manifest.
+   */
+  static merge(fieldsToIgnore: string[], webManifestUrl: URL
+      , webManifest: WebManifestJson, oldTwaManifest: TwaManifest): void {
+    let shortcuts: ShortcutInfo[] = [];
+    let name;
+    let launcherName;
+    let display;
+    let themeColor;
+    let backgroundColor;
+    let fullStartUrl: URL;
+    let startUrl;
+    let icon: WebManifestIcon | null = null;
+    let maskableIcon: WebManifestIcon | null = null;
+    let monochromeIcon: WebManifestIcon | null = null;
+
+    const packageId = oldTwaManifest.packageId;
+    const host = oldTwaManifest.host;
+    const navigationColor = oldTwaManifest.navigationColor.hex();
+    const navigationColorDark = oldTwaManifest.navigationColorDark.hex();
+    const navigationDividerColor = oldTwaManifest.navigationDividerColor.hex();
+    const navigationDividerColorDark = oldTwaManifest.navigationDividerColorDark.hex();
+    const appVersionName = oldTwaManifest.appVersionName;
+    const signingKey = oldTwaManifest.signingKey;
+    const splashScreenFadeOutDuration = oldTwaManifest.splashScreenFadeOutDuration;
+    const enableNotifications = oldTwaManifest.enableNotifications;
+    const twaManifestShortcuts = shortcuts;
+    const twaManifestWebManifestUrl = oldTwaManifest.webManifestUrl!.toString();
```

Comment:
These are just the variables that we're passing straight into the new manifest's constructor?

I think having variables for these is just noise, just do:

```
    const twaManifest = new TwaManifest({
      packageId: oldTwaManifest.packageId,
```